satpy/tests/reader_tests/test_aws_l1b.py

```diff
@@ -0,0 +1,179 @@
+"""Tests for aws l1b filehandlers."""
```

Comment:
Naming: AWS -> MWS.

Reply:
MWS is another sensor onboard EPS-SG-A. MWR is the name, you are right!